src/test/java/org/prebid/server/bidder/conversant/ConversantBidderTest.java

```diff
@@ -449,8 +506,11 @@ public void makeBidsShouldReturnErrorIfResponseBodyCouldNotBeParsed() {
 
         // then
         assertThat(result.getErrors()).hasSize(1);
-        assertThat(result.getErrors().get(0).getMessage()).startsWith("Failed to decode: Unrecognized token");
         assertThat(result.getErrors().get(0).getType()).isEqualTo(BidderError.Type.bad_server_response);
```

Comment:
redundant

Reply:
You mean we need to keep back the error message?

Reply:
agreed.